streams/src/test/java/org/apache/kafka/streams/state/internals/AbstractRocksDBSegmentedBytesStoreTest.java

```diff
@@ -512,19 +512,24 @@ private Bytes serializeKey(final Windowed<String> key) {
         final StateSerdes<String, Long> stateSerdes = StateSerdes.withBuiltinTypes("dummy", String.class, Long.class);
         while (iterator.hasNext()) {
             final KeyValue<Bytes, byte[]> next = iterator.next();
+            final KeyValue<Windowed<String>, Long> deserialized;
```

Comment:
Not a huge fan of this particular "clean up". We're trading a tightly defined scope for de-duplicating the trivial line `results.add(deserialization)`. It doesn't seem like a good trade to me.

Reply:
I don't feel "strong" about it -- it was an IntelliJ warning/suggestion to "share common code".